### PR TITLE
Adds a flag to prevent database (& dotenv presumptions)

### LINKED ISSUE #11: Support preventing the database setup
I noted an issue with the `updateSiteEnvironmentFile` when using no environment file.

This should be investigated, as I'm not sure if it's a bug (I did try adding these to my test projects when deploying)  so not sure if this is an issue on all first setups for a site.

A solution is to add a flag to prevent this behaviour:
```
{--no-database : Avoid creating a database.}
```

### PR DESCRIPTION
**Pull request by Illizian**
Closes #11

### CODE REVIEW

**Comment by Illizian** [comment]
I just wanted to check on the destroy command to ensure this behaviour doesn't need replicating to both commands and would cause dependencies, but it iterates across existing databases within the destroy command so we should be good.

File: app/Commands/DeployCommand.php [hunk 1]
```diff
@@ -32,7 +32,9 @@ class DeployCommand extends Command
         {--edit-env=* : The colon-separated name and value that will be added/updated in the site\'s environment, e.g. "MY_API_KEY:my_api_key_value".}
         {--scheduler : Setup a cronjob to run Laravel\'s scheduler.}
         {--no-quick-deploy : Create your site without "Quick Deploy".}
-        {--no-deploy : Avoid deploying the site.}';
+        {--no-deploy : Avoid deploying the site.}
+        {--no-isolation : Avoid site isolation.}
```
Comment (ryangjchandler) [inline app/Commands/DeployCommand.php]: This flag shouldn't be here.
```suggestion
```

**Review by ryangjchandler — CHANGES_REQUESTED** [review]

File: app/Commands/DeployCommand.php [hunk 2]
```diff
@@ -32,7 +32,8 @@ class DeployCommand extends Command
         {--edit-env=* : The colon-separated name and value that will be added/updated in the site\'s environment, e.g. "MY_API_KEY:my_api_key_value".}
         {--scheduler : Setup a cronjob to run Laravel\'s scheduler.}
         {--no-quick-deploy : Create your site without "Quick Deploy".}
-        {--no-deploy : Avoid deploying the site.}';
+        {--no-deploy : Avoid deploying the site.}
+        {--no-database : Avoid creating a database.}';
```
Comment (ryangjchandler) [inline app/Commands/DeployCommand.php]:
```suggestion
        {--no-db : Avoid creating a database.}';
```

File: app/Commands/DeployCommand.php [hunk 3]
```diff
@@ -50,7 +51,9 @@ public function handle(Forge $forge)
 
         $site = $this->findOrCreateSite($server);
 
-        $this->maybeCreateDatabase($server, $site);
+        if (!$this->option('no-database')) {
```
Comment (ryangjchandler) [inline app/Commands/DeployCommand.php]:
```suggestion
        if (! $this->option('no-db')) {
```

**Review by ryangjchandler — CHANGES_REQUESTED** [review]

**Review by ryangjchandler — APPROVED** [review]

**Pull request merged**